### PR TITLE
Integrate JSCS linter with Webpack using `jscs-loader`

### PR DESCRIPTION
**Pull request by edmondko**
I've integrated the JSCS Linter with webpack using this loader: https://github.com/unindented/jscs-loader; Integrating the JSCS in your text-editor or using it with CLI will improve the code consistency across all projects. I've included the airbnb preset, but it can be modified as we wish.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -6,9 +6,12 @@ module.exports = {
   },
   devtool: 'source-map',
   module: {
+    preloaders: [
+      { test: /\.js$/, loader: 'jscs-loader', exclude: /\/(spec|node_modules)/ }
```
Comment (chris-kobrzak) [inline webpack.config.js]: I'd strip the "-loader" bit to keep it more concise and consistent with the rest of the code

**Comment by chris-kobrzak** [comment]
Issue #5

File: .jscsrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "esnext": true,
+    "preset": "airbnb",
```
Comment (chris-kobrzak) [inline .jscsrc]: The trailing comma is currently breaking JSCS when you try running it from command line: `./node_modules/jscs/bin/jscs src/`

File: webpack.config.js (hunk 1 above)
Comment (edmondko) [inline webpack.config.js]: I think it's better to have the -loader part because it's the loader that is dependent on the 'jscs' package.

**Comment by chris-kobrzak** [comment]
https://github.com/whichdigital/webpack-react-workflow/issues/5 has now been merged so closing this PR.

**Pull request closed without merging**